### PR TITLE
Small edits.   

### PR DESCRIPTION
**Pull request by paulsheldrake**
Location could be less US specific

### CODE REVIEW

**Comment by populist** [comment]
I don't think having a specific "state" drop-down is going to generate a lot of statistically significant insights. Urban vs suburban vs rural might be more interesting.

**Comment by paulsheldrake** [comment]
@populist  That's a good point.   I've updated but I'm not sure on the best wording to use.  Any suggestions?

File: SURVEY.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We need to figure out pagination, order, pacing, and reasonable length, in addit
 ## How do you help build the internet?
 
 ### Position
-    What role do you play? (wording)
+    What role(s) do you play? (wording)
```
Comment (nstielau) [inline SURVEY.md]: @paulsheldrake if you can't think of any better work either, can you just remove the '(wording)' and we'll call it good?  Thx

File: SURVEY.md [hunk 2]
```diff
@@ -32,9 +32,13 @@ We need to figure out pagination, order, pacing, and reasonable length, in addit
     () 200+ employees
 
 ### Location
-    If you live in the US, please choose your state. |v| Dropdown of States |
 
-    If you live outside of the US, please choose your country. |v| Dropdown of Countries |
+    Please choose your country. |v| Dropdown of Countries |
+    
+    Please choose area(?  not sure what the best term is)
```
Comment (drnikko) [inline SURVEY.md]: Can we update this to "Which word best describes the area where you live and work?"

**Comment by nstielau** [comment]
Thanks @paulsheldrake.  A few pieces of feedback and we'll incorporate.  

Also, please add yourself to CONTRIBUTORS file if you'd like!

**Comment by drnikko** [comment]
@paulsheldrake do you want to fix these, or we can merge and come back to them later.

**Pull request merged**